contribute/index.md

```diff
@@ -31,6 +31,7 @@ Presently, there are two ways in which you can contribute:
 
     - Make sure you follow the [ROOT coding conventions]({{'/contribute/coding_conventions' | relative_url }}) in your code
     - Make sure you provide a set of tests for your feature/bug fix
+    - Make sure your git commits can be added to ROOT's master branch with a "rebase and merge", i.e. by fast-forwarding the master branch: ROOT tries to avoid merge commits as much as possible
```

Comment:
Reader might wonder 'why' ...

Reply:
any suggestion to improve the text?

Reply:
(part of) the text in NoSY (No switch-yard) was intended as that explanation (maybe it failed :) ).

Reply:
I inserted the line of the NoSY page that explained why we don't like merge commits here.

Reply:
thanks.